conan/internal/model/conan_file.py

```diff
@@ -353,8 +353,27 @@ def generators_path(self) -> Path:
         assert self.generators_folder is not None, "`generators_folder` is `None`"
         return Path(self.generators_folder)
 
-    def run(self, command, stdout=None, cwd=None, ignore_errors=False, env="", quiet=False,
+    def run(self, command: str, stdout=None, cwd=None, ignore_errors=False, env="", quiet=False,
```

Comment:
I have added an explicit `str` here, because there might be some confussion as to whether this should be an array of arguments or a full string, because Python's own api has both variants in different methods, and this might help clarify what we expect